scrapers/TheScoreGroup.yml

```
@@ -18,7 +19,7 @@ galleryByURL:
 xPathScrapers:
   sceneScraper:
     scene:
-      Title: //section[@id="videos_page-page"]/div[@class="row"]/div/h2/text()
+      Title: //main/div/section[1]/div[@class="row"]/div/h1/text()
```

there is no ends with isnt the contains enough though?
```suggestion
      Title: //section[contains(@id,"_page-page")]/div[@class="row"]/div/h1/text()
```